bbk-backend/BBK.API/Data/Models/Recipe.cs

```diff
@@ -0,0 +1,23 @@
+﻿using BBK.API.Contracts.Responses;
+
+namespace BBK.API.Data.Models;
+
+// TODO: this is an example model only
+public class Recipe
+{
+    public int Id { get; set; }
+    public required string Name { get; set; }
+    public required string Description { get; set; }
+}
+
+public static class RecipeExtensions
+{
+    public static RecipeResponse ToResponse(this Recipe model)
```

Comment:
Shouldn't this be an interface instead?
Perhaps I'm missing something but I don't see a point in this being an extension